README.md

```diff
@@ -36,3 +36,11 @@ vault write secret/reports/deploy public=@ssh/id_rsa.pub private=@ssh/id_rsa
 ```
 
 4. Remove the ssh keys with `rm -r ssh`
+
+### Populating static documentation volume
+The `documentation` folder contains scripts needed for populating the static documentation folder from a dropbox 
+folder. 
+Usage:
+1. `./build.sh` - Builds a docker image and pushes it to the vimc private docker registry. Runs on TeamCity.
```

Comment:
If this build is running, could you please configure the teamcity hook (see montagu-ci) so that it phones back here with the status?